gosdk/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/GoogleContainerTools/kpt-functions-sdk/gosdk
```

Comment:
Do we need this file?

Reply:
Mistakenly commited WIP dir. :) Great catch.